server/odc-service/src/main/java/com/oceanbase/odc/service/regulation/ruleset/RulesetService.java

```diff
@@ -57,7 +61,27 @@ public Ruleset detail(@NonNull Long id) {
         return ruleset;
     }
 
+    @Transactional(rollbackFor = Exception.class)
+    public Ruleset create(@NonNull Ruleset ruleset) {
```

Comment:
Lack of unit tests.

Reply:
It lacks unit tests as this was a WIP PR when you were reviewing. Now I added the unit cases, thanks for your reminder.